src/app/shared/money-input/money-input.component.css

```diff
@@ -0,0 +1,32 @@
+input {
+  text-align: center;
+  background: #ffffff;
+  border: 1px solid #240b18;
+  box-sizing: border-box;
+  border-radius: 5px;
+  width: 164px;
+  height: 58px;
+  font-size: 24px;
+}
+
+input[type='number']::-webkit-inner-spin-button,
+input[type='number']::-webkit-outer-spin-button {
+  display: none;
+}
+
+button {
+  background: #7e7c90;
+  width: 44px;
+  height: 34px;
+  font-size: 18px;
+
+  justify-content: center;
```

Comment:
How does this property work? It may only work for `display: flex;` or `display: grid;` as far as I know